axelrod/tests/strategies/test_stalker.py

```diff
@@ -21,26 +21,39 @@ class TestStalker(TestPlayer):
     }
 
     def test_strategy(self):
+        
         # Start with cooperation
         self.first_play_test(C)
 
         # current_average_score > very_good_score
-        self.responses_test([D], [C] * 2 + [D] * 4, [D] * 2 + [C] * 4)
```

Comment:
@alajara I've opened https://github.com/alajara/Axelrod/pull/4 to this branch.

It include the test that's missing here. It was a bit tricky to figure out how to get the `current_average > very_good_score` and I do that by tricking the player to defecting by lying to it about the length of the Match. 

Note that that PR also include a merge from master, if you click merge on the PR, it'll automatically appear here and should be good to go.